compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -648,9 +648,11 @@ class OMR_EXTENSIBLE CodeGenerator
 
    // high precision timer can be converted into millisecond timer or nanosecond timer through appropriate
    // division or multiplication (respectively). For example, 390 high precision timer provides accuracy to
-   // 1/8 of a microsecond so converters are 8000 and 125 respectively (multiplying the two should give you 1000000)
+   // 1/2048 of a microsecond so, to convert maximum-precision timer to
+   // - milliseconds, divide maximum-precision timer by 2,048,000
+   // - nanoseconds, multiply maximum-precision timer by 125/256
    int64_t getCurrentTimeMillisDivisor() { return 0; } // no virt
-   int64_t getNanoTimeMultiplier() { return 0; } // no virt
```

Comment:
There are no uses *now* but I feel like this is platform-specific information and it belongs to OMR. In addition, OMR has other mentions of these constants in port library; I would prefer not to smear this over two projects.